organization/src/main/kotlin/com/cosmotech/organization/service/OrganizationServiceImpl.kt

```diff
@@ -126,6 +135,14 @@ class OrganizationServiceImpl(
     if (organization.security != null && existingOrganization.security == null) {
```

Comment:
remove and prevent modification of security. security modification should be done in endpoint security